server/adaptors/integrations/__data__/repository/apache/apache-1.1.0.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "apache",
```

Comment:
Fixed the issue of 
```
Property keys must be doublequoted json(528)
```

Reply:
It works because of #1732, just the file extensions need to be changed per #1816